tools/xcodeproj_shims/swiftc-stub.sh

```diff
@@ -8,7 +8,7 @@ if [[ $# -eq 1 && $1 == "-v" ]]; then
 fi
 
 write_output_files() {
-    cat $1 | python $(dirname "$0")/print_json_leaf_nodes | xargs touch
+    cat $1 | python3 $(dirname "$0")/print_json_leaf_nodes | xargs touch
```

Comment:
This probably isn't a concern for older systems

Reply:
Since the Xcode 14 will only support macOS 12.4 + which doesn't come with python2 anymore, I guess it's OK to do so